vault/identity_store_oidc.go

```diff
@@ -747,13 +791,21 @@ func (i *IdentityStore) pathOIDCGenerateToken(ctx context.Context, req *logical.
 		return nil, err
 	}
 
+	retResp := &logical.Response{}
+	expiry := role.TokenTTL
+	if expiry > key.VerificationTTL {
+		expiry := key.VerificationTTL
```

Comment:
```suggestion
		expiry = key.VerificationTTL
```

Reply:
I think this should be a `=` instead of a `:=`. `:=` would scope the expiry assignment to the if block.

Reply:
Nice catch!

Reply:
Nice catch indeed!